Rakefile

```diff
@@ -150,144 +171,15 @@ namespace :playground do
     sh 'mkdir SwiftGen.playground/Resources'
   end
   task :images do
-    xcrun %Q(actool --compile SwiftGen.playground/Resources --platform iphoneos --minimum-deployment-target 7.0 --output-format=human-readable-text UnitTests/fixtures/Images.xcassets)
+    xcrun(%Q(actool --compile SwiftGen.playground/Resources --platform iphoneos --minimum-deployment-target 7.0 --output-format=human-readable-text UnitTests/fixtures/Images.xcassets), task)
   end
   task :storyboard do
-    xcrun %Q(ibtool --compile SwiftGen.playground/Resources/Wizard.storyboardc --flatten=NO UnitTests/fixtures/Storyboards-iOS/Wizard.storyboard)
+    xcrun(%Q(ibtool --compile SwiftGen.playground/Resources/Wizard.storyboardc --flatten=NO UnitTests/fixtures/Storyboards-iOS/Wizard.storyboard), task)
   end
   task :strings do
-    xcrun %Q(plutil -convert binary1 -o SwiftGen.playground/Resources/Localizable.strings UnitTests/fixtures/Localizable.strings)
+    xcrun(%Q(plutil -convert binary1 -o SwiftGen.playground/Resources/Localizable.strings UnitTests/fixtures/Localizable.strings), task)
   end
 
   desc "Regenerate all the Playground resources based on the test fixtures.\nThis compiles the needed fixtures and place them in SwiftGen.playground/Resources"
   task :resources => %w(clean images storyboard strings)
 end
-
-
-
-## [ Release a new version ] ##################################################
-
-namespace :release do
```

Comment:
I see the `feature/rake` branch in `Eve` but no PR, you don't consider it ready yet?